backend/routers/database_setup.py

```
@@ -0,0 +1,19 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+from .. import models, database
+
+router = APIRouter()
+
+@router.get("/init_db")
+def init_db(db: Session = Depends(database.get_db)):
+    """Initialize the database with all tables"""
+    try:
+        models.Base.metadata.create_all(bind=database.engine)
+        return {"message": "Database initialized successfully"}
```

Ruff (0.13.3)

Consider moving this statement to an `else` block

(TRY300)